hexrd/xrdutil/phutil.py

```diff
@@ -449,7 +452,7 @@ def _infer_eHat_l(panel):
 
     eHat_l_dict = {
         'TARDIS': -ct.lab_x.reshape((3, 1)),
-        'PXRDIP': ct.lab_y.reshape((3, 1))
+        'PXRDIP': -ct.lab_x.reshape((3, 1))
```

Comment:
Maybe we can get rid of this function entirely if `PXRDIP` and `TARDIS` are the same.

Reply:
I think the FIDDLE instrument might have to be treated separately. I'm not 100% about that yet, but it's possible. So I'd keep the cases for now. We can remove in a future PR if the FIDDLE instrument doesn't need a separate case.